docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -6,11 +6,15 @@ This is a list of things to do if you want to update from a previous version val
  - take a look at your custom files to see if there are some changes (e.g. `localConfig.js`, `proxy.properties`)
  - Follow the instructions, in order
 
+## Migration from 2019.03.00 to 2020.01.00
+
+- The translations file extension has been changed into JSON. Now translation files has been renamed from `data.<locale>` to `data.<locale>.json`. This change makes the `.json` extension mandatory for all translation files. This means that depending projects with custom translation files should be renabled in the same name. E.g. `data.it-IT` have to be renamed as `data.it-IT.json`
```

Comment:
should we link the issue, or the pr like :
for more details see #....

Reply:
Sure, I'd like to wait to merge this PR. I want to test if georchestra/mapstore2-georchestra#126 is solved applying these changes